R/class_mrgmod.R

```diff
@@ -100,6 +100,47 @@ check_globals <- function(x,cmt) {
   return(invisible(NULL))
 }
 
+check_cpp_dot <- function(env, x) {
+  # Vast majority of models will just return
+  if(is.null(env[["cpp_dot"]])) return(NULL)
+  cpp_dot <- setdiff(
+    env[["cpp_dot"]], 
+    x@envir[["MRGSOLVE_CPP_DOT_SKIP"]]
```

Comment:
One of the tests specifies multiple `MRGSOLVE_CPP_DOT_SKIP ` values as comma-delimited.  Is this spot supposed to account for that?

Reply:
Thanks. Yes, this is where that should get handled, but does not.